@here/harp-datasource-protocol/package.json

```diff
@@ -27,7 +27,8 @@
         "@here/harp-geometry": "^0.10.0",
         "@here/harp-geoutils": "^0.10.0",
         "@here/harp-lines": "^0.10.0",
-        "@here/harp-utils": "^0.10.0"
+        "@here/harp-utils": "^0.10.0",
+        "@here/harp-materials": "^0.10.0"
```

Comment:
Datasource protocol should't depend on "renderer" part of code, in particular `harp-materials`.
(example: we don't want shaders and stuff in decoder code  -minor issue, but an issue).

I know, this is for `LineCaps` definition.

I know our package architecture is mess, but don't make it worse...

All the code around "implements" features defined in `datasource-protocol` not the other way around, so please move `LineCaps` do `datasource-protocol` and reverse the dependency or ... even duplicate it as
* "material" `LineCaps` should be actually an enum 
* "theme" `LineCaps` is "string enum" as defined by user in theme file
* "string" to "enum" should be explicit mapping with sane fallback - what if user will pass `"FOO"` as caps ? at first look something will explode somewhere

Reply:
Done.